scripts/buildProtocol.ts

```diff
@@ -113,7 +113,7 @@ class DeclarationsWalker {
     } 
 }
 
-function generateProtocolFile(protocolTs: string, typeScriptServicesDts: string): string {
+function generateProtocolFile(outputFile: string, protocolTs: string, typeScriptServicesDts: string) {

```

Comment:
While I think "writeProtocolFile" might be clearer, that explanation makes sense.